Build/Build.cs

```diff
@@ -97,19 +102,32 @@ public override ISbomGeneratorBuilder ConfigureSbomGenerationInfoBuilder (Soluti
 
   public override void ConfigureProjects (ProjectsBuilder projects)
   {
-    var normalTestConfiguration = new TestConfiguration(
-        DefaultTestExecutionRuntimeFactory.Instance,
-        TestMatrices.Single(e => e.Name == "NormalTestMatrix"),
+    [CanBeNull]
+    static TestConfiguration CreateTestConfiguration (
+        [CanBeNull] TestMatrix testMatrix,
+        ITestExecutionRuntimeFactory testExecutionRuntimeFactory,
+        ImmutableArray<ITestExecutionWrapper> testExecutionWrappers)
+    {
+      return testMatrix != null
+          ? new TestConfiguration(testExecutionRuntimeFactory, testMatrix, testExecutionWrappers)
+          : null;
+    }
```

Comment:
Normally, we try to put the local functions after the method's body. Of course, here we would need to put it after the local variables setup and before the project configration because otherwise it would be completely unreadable. I can understand why you want to keep it up top and thus at least somewhat consistent, though